test/Makefile

```diff
@@ -1,14 +1,14 @@
 export LC_ALL := C
 images_ch := chtest/Build \
-             $(sort $(wildcard ./Build.*)) \
-             $(sort $(wildcard ./Dockerfile.*)) \
-             $(sort $(wildcard ./Docker_Pull.*))
+		 $(sort $(wildcard ./Build.*)) \
+		 $(sort $(wildcard ./Dockerfile.*)) \
+		 $(sort $(wildcard ./Docker_Pull.*))
```

Comment:
Spurious tab characters introduced in several places. The only place we want ASCII 9 in the file is in Make recipes where it is required, and there only once.